userdocs/src/gitops-quickstart.md

```diff
@@ -289,6 +276,47 @@ Congratulations to your gitopsed cluster on EKS!
 
 ## Advanced setups
 
+### Creating a cluster with Gitops in one command
+
+If you already have experience with Gitops in eksctl you might want to create a cluster with a Gitops setup at once. To
+do that, simply add the Gitops configuration to the cluster's config file like in the example below:
+
+```YAML
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-21
+  region: eu-north-1
+
+# other cluster config ...
+
+git:
+  repo:
+    url: "git@github.com:myorg/cluster-21.git"
+    branch: master
+    fluxPath: "flux/"
+    user: "gitops"
+    email: "gitops@myorg.com"
+  operator:
+    namespace: "flux"
+    withHelm: true
+  bootstrapProfiles:
```

Comment:
Reminder: needs to be updated

```suggestion
  bootstrapProfile:
```

Reply:
Thanks, there were a bunch of these in the docs